src/sentry/api/endpoints/organization_events.py

```diff
@@ -13,10 +13,47 @@
 from sentry.api.paginator import GenericOffsetPaginator
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models.event import SnubaEvent
+from sentry.api.utils import parse_stats_period
 from sentry.models import OrganizationMember, OrganizationMemberTeam, Project, ProjectStatus
+from sentry.search.utils import parse_datetime_string, InvalidQuery
 from sentry.utils.snuba import raw_query
 
 
+MIN_STATS_PERIOD = timedelta(hours=1)
+MAX_STATS_PERIOD = timedelta(days=90)
+
+
+class InvalidParams(Exception):
+    pass
+
+
+def get_date_range_from_params(params):
+    # Returns (start, end) or raises an `InvalidParams` exception
+    now = timezone.now()
+
+    end = now
+    start = now - MAX_STATS_PERIOD
+
+    stats_period = params.get('statsPeriod')
+    if stats_period is not None:
+        stats_period = parse_stats_period(params['statsPeriod'])
```

Comment:
Nit: why not use the `stats_period` var here?

```suggestion
        stats_period = parse_stats_period(stats_period)
```